packages/kbn-optimizer/limits.yml

```diff
@@ -113,6 +113,6 @@ pageLoadAssetSize:
   uiActionsEnhanced: 38494
   urlDrilldown: 30063
   indexPatternEditor: 19123
-  indexPatternFieldEditor: 34448
+  dataViewFieldEditor: 20000
```

Comment:
Also curious on this one: only spot we seem to be changing something other than the naming.

Reply:
its a better reflection of the size of the plugin bundle size